2003-12-12 05:00    diff -url lms.orig/index.php lms/index.php    Page 1


--- lms.orig/index.php	Fri Dec 12 04:58:02 2003
+++ lms/index.php	Fri Dec 12 05:00:20 2003
@@ -53,14 +53,14 @@
 									
 // Define directories and configuration vars
 
-$_SYSTEM_DIR = (! $_CONFIG[directories]['sys_dir'] ? getcwd() : $_CONFIG[directories]['sys_dir']);
-$_BACKUP_DIR = (! $_CONFIG[directories]['backup_dir'] ? $_SYSTEM_DIR.'/backups' : $_CONFIG[directories]['backup_dir']);
+$_SYSTEM_DIR = (! $_CONFIG[directories]['sys_dir'] ? '/usr/share/lms' : $_CONFIG[directories]['sys_dir']);
+$_BACKUP_DIR = (! $_CONFIG[directories]['backup_dir'] ? '/var/lib/lms/backups' : $_CONFIG[directories]['backup_dir']);
 $_LIB_DIR = (! $_CONFIG[directories]['lib_dir'] ? $_SYSTEM_DIR.'/lib' : $_CONFIG[directories]['lib_dir']);
 $_MODULES_DIR = (! $_CONFIG[directories]['modules_dir'] ? $_SYSTEM_DIR.'/modules' : $_CONFIG[directories]['modules_dir']);
-$_SMARTY_DIR = (! $_CONFIG[directories]['smarty_dir'] ? (is_readable('/usr/share/php/smarty/libs/Smarty.class.php') ?  '/usr/share/php/smarty/libs' : $_LIB_DIR.'/Smarty' ) : $_CONFIG[directories]['smarty_dir']);
-$_SMARTY_COMPILE_DIR = (! $_CONFIG[directories]['smarty_compile_dir'] ? $_SYSTEM_DIR.'/templates_c' : $_CONFIG[directories]['smarty_compile_dir']);
-$_SMARTY_TEMPLATES_DIR = (! $_CONFIG[directories]['smarty_templates_dir'] ? $_SYSTEM_DIR.'/templates' : $_CONFIG[directories]['smarty_templates_dir']);
-$_ADODB_DIR = (! $_CONFIG[directories]['adodb_dir'] ? (is_readable('/usr/share/adodb/adodb.inc.php') ?  '/usr/share/adodb' : $_LIB_DIR.'/adodb' ) : $_CONFIG[directories]['adodb_dir']);
+$_SMARTY_DIR = (! $_CONFIG[directories]['smarty_dir'] ? '/usr/share/pear/Smarty' : $_CONFIG[directories]['smarty_dir']);
+$_SMARTY_COMPILE_DIR = (! $_CONFIG[directories]['smarty_compile_dir'] ? '/var/lib/lms/templates_c/' : $_CONFIG[directories]['smarty_compile_dir']);
+$_SMARTY_TEMPLATES_DIR = (! $_CONFIG[directories]['smarty_templates_dir'] ? $_SYSTEM_DIR.'/templates/' : $_CONFIG[directories]['smarty_templates_dir']);
+$_ADODB_DIR = (! $_CONFIG[directories]['adodb_dir'] ? '/usr/share/pear/adodb/' : $_CONFIG[directories]['adodb_dir']);
 $_TIMEOUT = (! $_CONFIG[phpui]['timeout'] ? 600 : $_CONFIG[phpui]['timeout']);
 $_FORCE_SSL = chkconfig($_CONFIG[phpui]['force_ssl']);
 
@@ -76,7 +76,7 @@
 
 $_DBTYPE = (! $_CONFIG['database']['type'] ? 'mysql' : $_CONFIG['database']['type']);
 $_DBHOST = (! $_CONFIG['database']['host'] ? 'localhost' : $_CONFIG['database']['host']);
-$_DBUSER = (! $_CONFIG['database']['user'] ? 'root' : $_CONFIG['database']['user']);
+$_DBUSER = (! $_CONFIG['database']['user'] ? 'mysql' : $_CONFIG['database']['user']);
 $_DBPASS = (! $_CONFIG['database']['password'] ? '' : $_CONFIG['database']['password']);
 $_DBNAME = (! $_CONFIG['database']['database'] ? 'lms' : $_CONFIG['database']['database']);
 
Only in lms: index.php~
--- lms/sample/lms.ini~	Sun Aug  8 02:54:14 2004
+++ lms/sample/lms.ini	Sun Aug  8 02:54:14 2004
@@ -166,7 +166,7 @@
 ; lib_dir.
 ; http://smarty.php.net
 
-; smarty_dir		= /usr/lib/php/Smarty
+; smarty_dir		= /usr/share/pear/Smarty
 
 ; Katalog kompilacji Smartyego. Miejsce gdzie Smarty psuje nasze
 ; templejty. Domy¶lnie to templates_c w katalogu sysdir.
@@ -182,7 +182,7 @@
 ; podkatalog adodb w lib_dir.
 ; http://php.weblogs.com/ADOdb
 ;
-; adodb_dir		= /usr/lib/php/adodb
+; adodb_dir		= /usr/share/pear/adodb
 
 ; ---------------------------------------------------------------------------
 ; Konfiguracja skryptu do powiadamiania - lms-notify
